app/src/main/java/com/sopt/now/presentation/main/friends/FriendFragment.kt

```diff
@@ -0,0 +1,129 @@
+package com.sopt.now.presentation.main.friends
+
+import android.content.DialogInterface
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.lifecycleScope
+import androidx.recyclerview.widget.ConcatAdapter
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.room.Room
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import com.sopt.now.R
+import com.sopt.now.data.database.FriendDatabase
+import com.sopt.now.data.model.FriendEntity
+import com.sopt.now.databinding.FragmentFriendBinding
+import com.sopt.now.presentation.main.friends.addfriend.AddFriendActivity
+import kotlinx.coroutines.launch
+
+class MainFragment : Fragment() {
+    val myProfileAdapter = MyProfileAdapter()
+    val friendAdapter = FriendAdapter(
+        onClicked = {
+            Toast.makeText(requireContext(), "클릭", Toast.LENGTH_SHORT).show()
+        },
+        onLongClicked = {
+            makeFriendDialog(it.id)
+            Toast.makeText(requireContext(), "롱클릭", Toast.LENGTH_SHORT).show()
+        }
+    )
+
+    //Todo. 더미데이터옮기기
```

Comment:
```suggestion
    // TODO 더미데이터옮기기
```
이런 식으로 적으면 나중에 찾기 편하답니둥